src/main/java/mate/academy/Main.java

```diff
@@ -6,17 +6,21 @@
 import mate.academy.service.AuthenticationServiceImpl;
 import mate.academy.service.OrderService;
 import mate.academy.service.OrderServiceImpl;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class Main {
+    private static final Logger logger = LogManager.getLogger();
+
     public static void main(String[] args) {
         AuthenticationService authenticationService = new AuthenticationServiceImpl();
-        User user;
+        User user = null;
         try {
             user = authenticationService.login("bob", "1234");
         } catch (AuthenticationException e) {
-            e.printStackTrace();
-            return;
+            logger.error("Authentication failed for username 'bob'. Reason: {}", e.getMessage());
         }
+        logger.info("Attempting to complete order for user ID '{}'.", user.getUserId());
```

Comment:
There is a potential null pointer exception here. If the authentication fails, `user` will remain null, and calling `user.getUserId()` will throw a `NullPointerException`. Consider adding a null check before attempting to use the `user` object.